tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -561,6 +564,69 @@ public function testAlterTableAutoIncrementIntToBigInt(
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testListTableColumnsOidConflictWithNonTableObject(): void
+    {
+        $wrappedConnection = $this->connection->getWrappedConnection();
+        assert($wrappedConnection instanceof ServerInfoAwareConnection);
+        if (version_compare($wrappedConnection->getServerVersion(), '10.0', '<')) {
+            self::markTestSkipped('Manually setting the Oid is not supported in 9.4');
+        }
```

Comment:
It appears we need to skip that test on Postgres 10 and 11 as well. #5955

Reply:
:( Sorry about that

Reply:
No worries, my comment was just FYI. 🙂